packages/block-library/src/gallery/edit.js

```diff
@@ -78,6 +81,13 @@ const linkOptions = [
 		value: LINK_DESTINATION_MEDIA,
 		noticeText: __( 'Media Files' ),
 	},
+	{
+		icon: fullscreen,
+		label: __( 'Expand on click' ),
+		value: LINK_DESTINATION_LIGHTBOX,
+		noticeText: __( 'Lightbox effect' ),
+		infoText: __( 'Scales the image with a lightbox effect' ),
```

Comment:
```suggestion
		infoText: __( 'Scale images with a lightbox effect' ),
```

I'm not a native English speaker, but it might be better to use the plural, as well as the other options.